merlin/models/tf/__init__.py

```diff
@@ -94,6 +94,7 @@
 )
 from .losses import LossType
 from .metrics.ranking import AvgPrecisionAt, NDCGAt, RecallAt, ranking_metrics
+from .models.benchmark import NCFModel
```

Comment:
I think we shouldn’t expose this directly in our public API. I think just importing benchmark is enough here, so users can use it by doing: `ml.benchmark.NCFModel(...)`